fork.Dockerfile

```diff
@@ -0,0 +1,28 @@
+# NOTE: The `latest` version of foundry does not include arm64 as a build platform,
```

Comment:
What's the idea behind this dockerfile? Is the idea just that it ships with compiled contracts and the deployment tools? If so, we should consider using a different name than `fork.Dockerfile` since the contents of this file don't really relate to using a forked anvil node. This said, I could be missing something.

Reply:
combined with original